Src/Couchbase.Linq/Extensions/BucketExtensions.cs

```diff
@@ -8,7 +9,8 @@ public static class BucketExtensions
     {
         public static IQueryable<T> Queryable<T>(this IBucket bucket)
         {
-            return EntityFilterManager.ApplyFilters(new BucketQueryable<T>(bucket));
+            //TODO refactor so ClientConfiguration is injectable
+            return EntityFilterManager.ApplyFilters(new BucketQueryable<T>(bucket, new ClientConfiguration()));
```

Comment:
Sounds good to me.  It will just require refactoring a bunch of the BeerSampleTests, they all use BucketExtensions I think.

If we're renaming stuff, I wanted to float something else by you.  When I wrote the EntityFilterManager stuff, my head was in EF world.  But in hindsight, I wonder if naming that stuff DocumentXXX instead of EntityXXX might be more appropriate in Couchbase land.  Thoughts?

Brant